gw_eccentricity/load_data.py

```diff
@@ -824,6 +843,11 @@ def load_sxs_catalogformat(**kwargs):
         to locate the strain file. This function will seek a file named
         `Strain_N{extrap_order}.h5` in the `data_dir`.
 
+    keep_memory: bool
+        If False, remove memory contribution from the waveform modes.
+        This will require metadata file to find t_relax which is used
+        to start the integration for computing memory contribution.
```

Comment:
Should it say what the default is?

Reply:
Earlier we decided that we are not going to mention the default for each kwarg here and instead refer to `load_data.get_load_waveform_defaults('SXSCatalog')` for getting the defaults. For example at the of of the do string under parameters, we refer to this function. The idea was to set the defaults for NR at a single place because some of these are shared by different waveform origins.

Reply:
Ok, I'll leave it to you, just make sure it would be easy for a reader to understand, and feel free to veto old decisions if they no longer make sense.

Reply:
I don't have a strong opinion about changing things as of now. But we can consider changing the current setup later, possibly in a separate PR.